gson/src/main/java/com/google/gson/internal/bind/ReflectiveTypeAdapterFactory.java

```diff
@@ -248,7 +291,12 @@ protected BoundField(String name, boolean serialized, boolean deserialized) {
       } catch (IllegalAccessException e) {
         throw new AssertionError(e);
       }
+      if (defaultTyping) {
+        out.endObject();
+      }
       out.endObject();
+
     }
+
```

Comment:
```suggestion
    }
```
These empty lines are probably not needed